pkg/lib/files/files.go

```diff
@@ -58,28 +109,38 @@ func IsFileOrDir(path string) bool {
 	return false
 }
 
-func IsDir(path string) bool {
+func IsDir(path string) error {
```

Comment:
What is the reason to make this return error? We can discuss. If we decide to leave it as returning error, maybe call it `CheckDir` or something (just because "is" might imply that it returns a boolean).